src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodExtraThrows.java

```diff
@@ -0,0 +1,112 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
+
+public class InputJavadocMethodExtraThrows {
+
+    /**
+     * Extra throws in javadoc it is ok.
+     * @param properties some value
+     * @throws IllegalArgumentException when argument is wrong // ok
+     * @throws NullPointerException indicates null was passed // ok
+     */
+    public InputJavadocMethodExtraThrows(String properties) {
+        if (properties.charAt(0) == 0) {
+            throw new IllegalArgumentException("cannot have char with code 0");
+        }
+        // here is NPE possible
```

Comment:
Actually, NPE will be not here, but in the first line of this method.

Reply:
extra statement is removed.